packages/toolkit/src/query/core/buildMiddleware/polling.ts

```diff
@@ -125,13 +132,15 @@ export const buildPollingHandler: InternalHandlerBuilder = ({
 
   function findLowestPollingInterval(subscribers: Subscribers = {}) {
     let lowestPollingInterval = Number.POSITIVE_INFINITY
-    for (const subscription of Object.values(subscribers)) {
-      if (!!subscription.pollingInterval)
+    for (let key in subscribers) {
```

Comment:
:speech_balloon: Saw that `findLowestPollingInterval` was showing up at the tail end of some of the perf capture flame charts. I don't expect this makes any real difference, but I don't think it _hurts_.